akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -26,20 +24,29 @@ import akka.persistence.SaveSnapshotFailure
 import akka.persistence.SaveSnapshotSuccess
 import akka.persistence.SnapshotProtocol
 import akka.persistence.journal.Tagged
-import akka.persistence.typed.DeleteEventsCompleted
-import akka.persistence.typed.DeleteEventsFailed
-import akka.persistence.typed.DeleteSnapshotsCompleted
-import akka.persistence.typed.DeleteSnapshotsFailed
-import akka.persistence.typed.DeletionTarget
-import akka.persistence.typed.EventRejectedException
-import akka.persistence.typed.SnapshotCompleted
-import akka.persistence.typed.SnapshotFailed
-import akka.persistence.typed.SnapshotMetadata
-import akka.persistence.typed.SnapshotSelectionCriteria
+import akka.persistence.query.{ EventEnvelope, PersistenceQuery }
+import akka.persistence.query.scaladsl.EventsByPersistenceIdQuery
+import akka.persistence.query.journal.leveldb.scaladsl.LeveldbReadJournal
+import akka.persistence.typed.{
+  DeleteEventsCompleted,
+  DeleteEventsFailed,
+  DeleteSnapshotsCompleted,
+  DeleteSnapshotsFailed,
+  DeletionTarget,
+  EventRejectedException,
+  PersistenceId,
+  SnapshotCompleted,
+  SnapshotFailed,
+  SnapshotMetadata,
+  SnapshotSelectionCriteria
+}
```

Comment:
Weird style for imports, using default intellij formatting somehow or is this what scalafmt wants today?

Reply:
I think that is what happens when they're slowly added by intellij and then when they hit a line length scalafmt does this